src/main/java/com/kuzmins/service/UserService.java

```diff
@@ -0,0 +1,64 @@
+package com.kuzmins.service;
+
+import com.kuzmins.model.tickets.Ticket;
+import com.kuzmins.model.tickets.TicketType;
+import com.kuzmins.model.users.User;
+import com.kuzmins.repositories.TicketRepository;
+import com.kuzmins.repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+@Transactional(readOnly = true)
+public class UserService{
+
+    @Value("${switcherForUpdateUserAndCreateTicket}")
+    private String switcherForUpdateUserAndCreateTicket;
+
+    private UserRepository userRepository;
+    private TicketRepository ticketRepository;
+
+    @Autowired
+    public UserService(UserRepository userRepository, TicketRepository ticketRepository) {
+        this.userRepository = userRepository;
+        this.ticketRepository = ticketRepository;
+    }
```

Comment:
instead of that you can use 1 annotation `@RequiredArgsConstructor` on class level

Reply:
added